### PR TITLE
feat: implement /ad page

### LINKED ISSUE #17: 🚀 Feature: Implement an Ad page
### Bug Report Checklist

- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/boston-ts-website/issues?q=is%3Aissue) and found none that matched my issue.

### Overview

The Philly JS club site has an `/ad` page used for promotional banners and the like. Let's add that here!

Screenshot of the Philly JS ad page

### Additional Info

_No response_

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
## PR Checklist

- [x] Addresses an existing open issue: fixes #17 
- [x] That issue was marked as [`status: accepting prs`](https://github.com/JoshuaKGoldberg/boston-ts-website/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)
- [x] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/boston-ts-website/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

Adds an `/ad` page hardcoded to April 2024, along with a logo asset.

### CODE REVIEW

File: app/components/AdLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+				{/* <path d="M1000 0H0V1000H1000V0Z" fill={foreground} />
+				<path d="M822 868H902.4V434.8H822V868Z" fill={background} />
+				<path
+					d="M546.22 869H627.22V732.2C627.22 699.8 646.42 681.8 678.82 681.8C707.62 681.8 721.42 698 721.42 726.8V869H802.42V721.4C802.42 648.8 762.22 608 706.42 608C671.02 608 642.82 624.2 627.22 653.6V435.8H546.22V869Z"
+					fill={background}
+				/>
+				<path
+					d="M234.85 868H318.85V775H347.65C461.05 775 530.05 716.2 530.05 617.8C530.05 519.4 461.65 460 347.65 460H234.85V868ZM318.85 701.2V533.8H345.25C412.45 533.8 443.65 561.4 443.65 617.8C443.65 673.6 413.65 701.2 345.25 701.2H318.85Z"
+					fill={background}
+				/> */}
```
Comment (git-megan) [inline app/components/AdLogo.tsx:43]: Delete the commented code - not needed

**Review by git-megan — COMMENTED** [review]
Styling looks great! Just remove the commented out code.

**Review by git-megan — CHANGES_REQUESTED** [review]
Oops clicked the button too quickly - please make changes

**Review by git-megan — APPROVED** [review]

**Comment by JoshuaKGoldberg** [comment]
@all-contributors please add @git-megan for maintenance.

> 🤖 Beep boop! This comment was added automatically by [all-contributors-auto-action](https://github.com/marketplace/actions/all-contributors-auto-action).
> Not all contributions can be detected from Git & GitHub alone. Please comment any missing contribution types this bot missed.
> ...and of course, thank you for contributing! 💙

**Pull request merged**